### PR TITLE
Don't crash with no networks

### PR DESCRIPTION
**Pull request by pcarranzav**
Currently, as connman-simplified returns an error, when the scan results in no networks found we crash.
This PR fixes this by catching the error, and showing an error message on the web UI.

### CODE REVIEW

**Comment by lorenzo-stoakes** [comment]
LGTM

File: src/app.coffee [hunk 1]
```diff
@@ -51,7 +51,11 @@ getIptablesRules = (callback) ->
 
 startServer = (wifi) ->
 	console.log('Getting networks list')
-	wifi.getNetworksAsync().then (list) ->
+	wifi.getNetworksAsync()
+	.catch (err) ->
+		throw err unless err.message == 'No WiFi networks found'
```
Comment (Page-) [inline src/app.coffee]: It makes me so sad they don't extend error with an `err.code`, meaning we have to do a string check which could change at any point :(

**Comment by Page-** [comment]
LGTM

File: src/app.coffee (hunk 1 above)
Comment (pcarranzav) [inline src/app.coffee]: Yeah, comparing strings is not very nice... I've already contributed to connman-simplified, so I might add this on a future PR...

**Pull request merged**